### DISCUSSION TITLE
Additional Secondary Firing Sequence

### DISCUSSION THREAD

**Discussion by deathreaperz** [Phobos-developers/Phobos#1237] [Ideas and feedback]
### Description

Asking for additional visual improvement for YR by enabling secondary firing sequence for infantry in water and in air.

### INI code

```ini
[SomeSequence]
SecondaryWetAttack=
SecondaryFireFly=
```

### Checklist

- [X] The suggestion is **NOT** related to functionality of Ares, HAres, Kratos or any other YR engine extension.
- [X] The suggested feature / enhancement is **NOT** implemented in Ares/Phobos yet and the existing possibilites are not viable or good enough to achieve the wanted result.
- [X] I agree to elaborate the details if requested and provide thorough testing if the feature is implemented.
- [X] I added a very descriptive title to this topic.
- [X] I used the GitHub search and read the topic list to find a similar topic and didn't find it.
- [X] I thought out how the feature / enhancement should work and described it in the most detailed and clear way.

**Comment by Starkku** [comment]
Good idea but technical implementation might be slightly complicated, sequence controls (the things that are parsed from INI) are stored in a fixed 42 item array, replacing that everywhere might end up being quite cumbersome.

**Reply by deathreaperz** [reply]
Awww, we hope it'll be dehardcoded soon.
